employees/tests/test_unit_report_viewset.py

```diff
@@ -431,11 +431,64 @@ def test_serializer_project_field_queryset_should_contain_only_projects_user_is_
         new_project.save()
         request = APIRequestFactory().post(path=self.url, data={"join": "join"})
         request.user = self.user
-        response = ReportList.as_view()(request)
+        response = ReportList.as_view()(request, **self.view_kwargs)
         project_field_choices = response.data["serializer"].fields["project"].choices.values()
         self.assertTrue(self.project.name in project_field_choices)
         self.assertFalse(new_project.name in project_field_choices)
 
+    def test_custom_report_list_view_should_redirect_to_another_month_if_month_switch_was_called_on_post(self):
+        request = APIRequestFactory().post(path=self.url, data={"date": "09-2020", "month-switch": "month-switch"})
```

Comment:
If https://github.com/Code-Poets/sheetstorm/pull/274 gets merged first, please remember to update tests, otherwise @Szymiks please remember to refactor those too.

Reply:
@Szymiks please be aware of this that I'm merging it now.